Supplement the doc on how to degrade v6.3.0 with `format-version: 2`. (#10538)
pingcap/docs: tikv-configuration-file.md

### `format-version` <span class="version-mark">New in v6.3.0</span>

> **Note:**
>

> > - After `format-version` is set to `2`, if you need to downgrade the TiKV cluster to a version earlier than v6.3.0, you should take following steps **before** downgrading:
> >   - Disable Raft Engine by setting `enable` to `false` and restart TiKV to make the configuration take effect.
> >   - Set `format-version` to `1` and enable Raft Engine by setting `enable` to `true`, then restart TiKV to make the configuration take effect.

Reviewer: ```suggestion
> After `format-version` is set to `2`, if you need to downgrade a TiKV cluster from v6.3.0 to an earlier version, take the following steps **before** the downgrade:
>
>   1. Disable Raft Engine by setting [`enable`](/tikv-configuration-file.md#enable-1) to `false` and restart TiKV to make the configuration take effect.
>   2. Set `format-version` to `1`.
>   3. Enable Raft Engine by setting `enable` to `true` and restart TiKV to make the configuration take effect.
```

### `format-version` <span class="version-mark">New in v6.3.0</span>

> **Note:**
>
> After `format-version` is set to `2`, if you need to downgrade a TiKV cluster from v6.3.0 to an earlier version, take the following steps **before** the downgrade:
>

> >   1. Disable Raft Engine by setting [`enable`](/tikv-configuration-file.md#enable-1) to `false` and restart TiKV to make the configuration take effect.
> >   2. Set `format-version` to `1`.
> >   3. Enable Raft Engine by setting `enable` to `true` and restart TiKV to make the configuration take effect.

Reviewer 2: ```suggestion
> 1. Disable Raft Engine by setting [`enable`](/tikv-configuration-file.md#enable-1) to `false` and restart TiKV to make the configuration take effect.
> 2. Set `format-version` to `1`.
> 3. Enable Raft Engine by setting `enable` to `true` and restart TiKV to make the configuration take effect.
```

> **Note:**
>

> > - This configuration item is only available when [`format-version`](#format-version-new-in-v630) >= 2.

Reviewer: ```suggestion
> This configuration item is only available when [`format-version`](#format-version-new-in-v630) >= 2.
```
